src/Data/Locales/pt_BR/messages.json

```diff
@@ -75,7 +75,7 @@
 	},
 	"about@sponsor":
 	{
-		"message": "Doar",
+		"message": "Me pague um café",
```

Comment:
It looks fine now, but I still think that it should be just "Support me" or "Donate" in languages other than English for the reason I've mentioned earlier.

![image](https://github.com/XFox111/PasswordGeneratorExtension/assets/28831743/b98a08b2-850b-451f-9a93-fd25d4cfc0c1)

Reply:
@XFox111 I made new updates see if it looks better now? If it doesn't look good, we'll keep it as before!

Reply:
@XFox111 I think we can keep this, because in Brazil there is something similar, https://mepagaumcafe.com.br/ where your link points to a website of the same model! Or if you want, you can make a page with links for donations and coffee is just an idea too!